util/crc32c_arm64.cc

```diff
@@ -35,6 +38,8 @@
   } while (0)
 #endif
 
+extern uint32_t pmull_runtime_flag;
```

Comment:
This is a C++ file. I don't see a reason using uint32_t to represent a boolean. Can we directly use a boolean? This also prevents the implicit casting when returning from crc32c_pmull_runtime_check().

Reply:
Yep, it makes sense, thanks.

The PR was updated:

**1. crc32c_test:**
```
yuqi@wls-arm-qc02:~/rocksdb$ ./crc32c_test
[==========] Running 4 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 4 tests from CRC
[ RUN      ] CRC.StandardResults
[       OK ] CRC.StandardResults (1 ms)
[ RUN      ] CRC.Values
[       OK ] CRC.Values (0 ms)
[ RUN      ] CRC.Extend
[       OK ] CRC.Extend (0 ms)
[ RUN      ] CRC.Mask
[       OK ] CRC.Mask (0 ms)
[----------] 4 tests from CRC (1 ms total)

[----------] Global test environment tear-down
[==========] 4 tests from 1 test case ran. (1 ms total)
[  PASSED  ] 4 tests.

```

**2. DB Benchmark:**
```
yuqi@wls-arm-qc02:~/rocksdb$ ./db_bench --benchmarks="crc32c"
Initializing RocksDB Options from the specified file
Initializing RocksDB Options from command-line flags
RocksDB:    version 6.13
Date:       Tue Sep 15 07:43:56 2020
CPU:        0 *
CPUCache:
Keys:       16 bytes each
Values:     100 bytes each (50 bytes after compression)
Entries:    1000000
Prefix:    0 bytes
Keys per prefix:    0
RawSize:    110.6 MB (estimated)
FileSize:   62.9 MB (estimated)
Write rate: 0 bytes/second
Read rate: 0 ops/second
Compression: Snappy
Compression sampling rate: 0
Memtablerep: skip_list
Perf Level: 1
WARNING: Assertions are enabled; benchmarks unnecessarily slow
------------------------------------------------
DB path: [/tmp/rocksdbtest-1004/dbbench]
crc32c       :       0.359 micros/op 2780130 ops/sec; 10859.9 MB/s (4096 per op)
```